core/ui/PageControls/new-journal.tid

```diff
@@ -16,5 +17,7 @@ description: {{$:/language/Buttons/NewJournal/Hint}}
 \end
 <$set name="journalTitleTemplate" value={{$:/config/NewJournal/Title}}>
 <$set name="journalTags" value={{$:/config/NewJournal/Tags}}>
+<$set name="journalText" value={{$:/config/NewJournal/Text}}>
```

Comment:
I think Jerome has already added a configuration panel to the master branch for the text field.